lib/shutil_custom/__init__.py

```diff
@@ -30,7 +30,7 @@ def special_file(fn):
 
     fdin = os.open(src, READ_FLAGS)
     fdout = os.open(dst, WRITE_FLAGS)
-    with os.fdopen(fdin) as fin, os.fdopen(fdout) as fout:
+    with open(fdin) as fin, open(fdout) as fout:
```

Comment:
A binary write mode would still be required for the output: `open(fdout, 'wb')`

Reply:
Let me do some reading on the default flags used with the builtin open.  If my assumption is correct the flags we are setting manually are the same as the defaults provided by the higher level open builtin.

Reply:
Sure thing, just let me know. Thanks!

Reply:
Afaik the built-in open is not an alias for io.open on Python < 3.

Reply:
> Since reading/writing here in raw binary though it makes no difference using `open` vs `io.open`

Docs for the built-in `open` on Python 2.7 doesn't say it accepts file descriptor objects like `io.open` does.
So IMO you need to use `io.open` regardless.

Reply:
So looks like all the flags can be removed and just use `open` in `rb` and `wb` modes.

Reply:
In any case I suggest using `io.open` for compatibility between the Python versions.

Reply:
@sharkykh you're right again, 2.7 open does not accept a fd.  I've all but abandoned python 2.7 since i converted my medusa to pure python 3.6

Reply:
@sharkykh  Sorry didn't finish my though.  You're right that in 2.7 `open` does not use accept file descriptors but the point is since the modes `'rb'` and `'wb'` are equivalent to the flags we use there is no need to even bother with `os.open` and file descriptors.

Reply:
Done